habinezadalvan/Banka
# #165645049 Fix validation bug and arrange files

**what does this PR do?**

- Fixes the validation bug so that a user can only view his transactions, and account details.
- Fixes the status codes bug
- Arranges the files into different folders
- Test all endpoints with the postman

**Description of Task to be completed.**

- Added validations for view transactions, create an account.
- Arranged the files

**How should this be tested manually?**

- clone repo
- cd banka
- run npm install
- run npm run start-dev
- go to the postman and test all endpoints and verify all validations
- To test run npm test

**Relative pivotal tracker story**

#165645049
